.github/workflows/build.yml

```diff
@@ -4,11 +4,11 @@ on:
   pull_request:
   workflow_dispatch:
     inputs:
-      isNightly:
+      isRelease:
         type: boolean
-        description: 'Is this a nightly build?'
+        description: 'Is this a release build?'
         required: false
-        default: true
+        default: false
```

Comment:
Default only works with `workflow_dispatch` apparently, the input will end up being an empty string if triggered by a push for example.